pkg/nfs/controllerserver.go

```diff
@@ -307,6 +315,52 @@ func (cs *ControllerServer) internalUnmount(ctx context.Context, vol *nfsVolume)
 	return err
 }
 
+func (cs *ControllerServer) copyFromVolume(ctx context.Context, req *csi.CreateVolumeRequest, dstVol *nfsVolume) error {
+	srcVol, err := getNfsVolFromID(req.GetVolumeContentSource().GetVolume().GetVolumeId())
+	if err != nil {
+		return err
+	}
+	srcPath := getInternalVolumePath(cs.Driver.workingMountDir, srcVol)
+	dstPath := getInternalVolumePath(cs.Driver.workingMountDir, dstVol)
+	klog.V(2).Infof("copy volume from volume %v -> %v", srcPath, dstPath)
+
+	var volCap *csi.VolumeCapability
+	if len(req.GetVolumeCapabilities()) > 0 {
+		volCap = req.GetVolumeCapabilities()[0]
+	}
```

Comment:
I am not sure if req capabilities apply for both src and dst volume internal mounting. I took this from here:
https://github.com/kubernetes-csi/csi-driver-nfs/blob/5edf03ce2f35507662efbc3447494b6ce56d1ec2/pkg/nfs/controllerserver.go#L120-L123